packages/app-cli/app/command-keymaps.ts

```diff
@@ -0,0 +1,47 @@
+import BaseCommand from './base-command';
+import app from './app';
+import { _ } from '@joplin/lib/locale';
+const { cliUtils } = require('./cli-utils.js');
+
+class Command extends BaseCommand {
+	public override usage() {
+		return 'keymaps';
+	}
+
+	public override description() {
+		return _('Displays the configured keyboard shortcuts.');
+	}
+
+	public override compatibleUis() {
+		return ['cli', 'gui'];
+	}
+
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any -- Old code before rule was applied
+	public override async action(_args: any) {
```

Comment:
Avoid using `@typescript-eslint/no-explicit-any` in new code: Consider instead defining an `Args` type near the top of the file (e.g. `interface Args {}`).